SPECS-EXTENDED/rust-cbindgen/rust-cbindgen.spec

```diff
@@ -0,0 +1,243 @@
+# Generated by rust2rpm 24
+# * tests are completely broken with Cython 3.0.0 on Fedora 39+
+Summary:        Tool for generating C bindings to Rust code
+Name:           rust-cbindgen
+Version:        0.24.3
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/mozilla/cbindgen
+Source:         %{url}/archive/refs/tags/v0.24.3.tar.gz
+# Below is a manually created tarball, no download link.
+# Note: the cbindgen-%%{version}-cargo.tar.gz file contains a cache created by capturing the contents downloaded into $CARGO_HOME.
+# To update the cache and config.toml run:
+#   tar -xf %{name}-%{version}.tar.gz
+#   cargo vendor > config.toml
+#   tar -czf %{name}-%{version}-cargo.tar.gz vendor/
+#
+Source1:        cbindgen-%{version}-cargo.tar.gz
+Source2:        cargo_config
+%global rustflags '-Clink-arg=-Wl,-z,relro,-z,now'
+%global _description %{expand:
+A tool for generating C bindings to Rust code.}
```

Comment:
Lets move these macros to the top.
Although, spec cleaner might not agree on this, our convention is to have them up-top.